src/components/MentorDetails.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link, useParams } from 'react-router-dom';
+import { fetchMentorDetails } from '../redux/slices/mentors/mentorDetailsSlice';
+import LoadingStatus from './ux/LoadingStatus';
+import ErrorStatus from './ux/ErrorStatus';
+
+const MentorDetails = () => {
```

Comment:
- [ ] When  I go to a mentor's details and click on the "Reserve" button, it just takes me back to the homepage.

![image](https://github.com/martinkarugaba/mentor_me_now_front_end/assets/57421823/1fde764e-ae9f-4f8d-9f5a-af351ec7ab7b)


Kindly, make it so this button takes your users to the "Reserve Mentor" page, so we can meet this particular requirement. 👍 

_tip: Just make the reserve button link to the /reserveMentor route._

Reply:
Hello @Meltrust  Thanks for taking the time to review this PR.

I want to point out that this PR has already been merged into the main branch, and we've requested the final assessment. 

Could you please advise on where we should implement the necessary modifications - should it be in the develop branch or directly in main?

Thank you for your quick response, and I'll be eagerly awaiting further instructions.